apps/btshell/src/cmd.c

```diff
@@ -104,17 +104,23 @@ parse_dev_addr(const char *prefix, const struct kv_pair *addr_types,
     if (!prefix) {
         name[0] = '\0';
     } else {
-        strcpy(name, prefix);
+        if (strlcpy(name, prefix, sizeof(name)) > sizeof(name)) {
```

Comment:
this (and others) should be  >=


also, XXX in 102 is no longer needed :)

Reply:
done :)